tests/manifest_test.py

```diff
@@ -30,8 +32,12 @@ def test_insert_order_does_not_matter(self):
         digest2 = hashing.Digest("test", b"efgh")
         item2 = manifest.FileManifestItem(path=path2, digest=digest2)
 
-        manifest1 = manifest.Manifest([item1, item2])
-        manifest2 = manifest.Manifest([item2, item1])
+        manifest1 = manifest.Manifest(
+            "test", [item1, item2], self._manifest_type
+        )
+        manifest2 = manifest.Manifest(
+            "test", [item2, item1], self._manifest_type
```

Comment:
nit: similar ambiguity for "test" throughout

Reply:
I'll change